credmark/cmf/types/token.py

```diff
@@ -23,8 +25,8 @@ def get_token_from_configuration(
     token_datas = [
         t for t in FUNGIBLE_TOKEN_DATA.get(chain_id, [])
```

Comment:
@leafyoung Shouldn't the FUNGIBLE_TOKEN_DATA be indexed into 2 dicts for looking up by symbol or address like we do for FIAT_CURRENCY_DATA_BY_ADDRESS? That would save us from looping through all the tokens.

Reply:
@leafyoung Do you want to do this? ^

Reply:
yup. I will do it. running the models test now.

Reply:
I made a commit before my flight across the pacific. I will be back online on Mon evening (CDT).